core/src/main/java/io/github/tcdl/msb/monitor/TopicStats.java

```diff
@@ -1,70 +0,0 @@
-package io.github.tcdl.msb.monitor;
-
-import java.time.Instant;
-
-/**
- * Immutable class that contains statistics for a topic.
- */
-class TopicStats {
```

Comment:
Also Instance statistics is mutable or not as we don't mention in JavaDoc for it